config/environments/development.rb

```diff
@@ -6,7 +6,7 @@
   # In the development environment your application's code is reloaded any time
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.enable_reloading = true
```

Comment:
NOTE: Even though raise_on_missing_translations was set to true in Rails 7.0, it did not raise exceptions in cases of missing translations.